tests/tasks/cleanup.yml

```diff
@@ -63,11 +62,13 @@
     /var/log/pacemaker/pacemaker.log
     /etc/yum.repos.d/packages-microsoft-com-*
     /tmp/*.j2
+    /tmp/mssql_data
+    /tmp/mssql_log
     /etc/systemd/system/multi-user.target.wants/mssql-server.service
   register: __mssql_cleanup_remove
   changed_when: "'removed' in __mssql_cleanup_remove.stdout"
 
 # On SQL Server 2017, 2019 the service remains after removing RPMs
 - name: Stop the mssql-server service  # noqa command-instead-of-module
-  command: systemctl stop mssql-server  # noqa ignore-errors
-  ignore_errors: true  # noqa no-changed-when
+  shell: systemctl stop mssql-server || true
+  changed_when: false
```

Comment:
Can you use the `service:` module or the `systemd:` module instead of `shell: systemctl ...`?

Reply:
mssql-server.service acts weirdly sometimes, and this is one of this. In the past, running this command was successful, but doing it with a `service` module failed because Ansible didn't find such a service. I'll check once again now.

Reply:
When stopping using the `sevrice` module, it fails with `Could not find the requested service mssql-server: host`. But this is required for a proper cleanup.